### PR TITLE
Move to netlify

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: worker/src/handlers/schedule.ts [hunk 1]
```diff
@@ -79,6 +79,9 @@ async function processQueue(): Promise<void> {
     );
     await KV.put(KV_KEY_CORE_ANALYTICS, JSON.stringify(core_analytics));
     queue.data = createQueueData();
+
+    // Trigger Netligy build
```
Comment (balloob) [inline worker/src/handlers/schedule.ts:83]:
```suggestion
    // Trigger Netlify build
```

File: worker/src/handlers/schedule.ts [hunk 2]
```diff
@@ -79,6 +79,9 @@ async function processQueue(): Promise<void> {
...
+    await fetch(process.env.NETLIFY, { method: "POST" });
```
Comment (balloob) [inline worker/src/handlers/schedule.ts:84]: What about

```suggestion
    await fetch(process.env.ANALYTICS_READY_URL, { method: "POST" });
```

**Review by balloob — APPROVED** [review]

File: worker/src/index.ts [hunk 3]
```diff
@@ -9,8 +8,6 @@ declare global {
 addEventListener("fetch", (event: FetchEvent) => {
   if (event.request.method === "POST") {
     event.respondWith(handlePost(event.request));
-  } else if (event.request.method === "GET") {
-    event.respondWith(handleGet(event.request));
```
Comment (bramkragten) [inline worker/src/index.ts:13]: We should respond with a 404 probably?

Comment (ludeeus) [reply worker/src/index.ts:13]: Yeah, should od that for everything, not just get

Comment (ludeeus) [reply worker/src/index.ts:13]: https://github.com/home-assistant/analytics.home-assistant.io/pull/73

**Pull request merged**